### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by sanlopez**

### CODE REVIEW

File: empiar/protocols/protocol_downloader.py [hunk 1]
```diff
@@ -52,9 +52,27 @@ def _defineParams(self, form):
...
+
+        line.addParam('doseInitial', params.FloatParam, default=0,
+                      label='Initial')
+
+        line.addParam('dosePerFrame', params.FloatParam, default=None,
+                      allowsNull=True,
+                      label='Per frame')
+
         # Parallel section defining the number of threads and mpi to use
         form.addParallelSection(threads=3, mpi=1)
```
Comment (pconesa) [inline empiar/protocols/protocol_downloader.py:70]: I think this line can go away.

**Review by pconesa — APPROVED** [review]
All fine from my side.

File: empiar/protocols/protocol_downloader.py (hunk 1 above)
Comment (sanlopez) [reply empiar/protocols/protocol_downloader.py:66]: Line removed :+1:

Comment (pconesa) [reply empiar/protocols/protocol_downloader.py:66]: Oh no!! I was referring to the addParallelSection

**Pull request merged**